script.js

```diff
@@ -1,4 +1,312 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+var deck = [];
+var playerHand = [];
+var computerHand = [];
+var playerScore;
+var computerScore;
+var displayPlayerHand = [];
+var displayComputerHand = [];
+var winner = "";
+var input;
+
+// Draw a card
+var drawCard = function (numCards, hand) {
+  for (var i = 0; i < numCards; i += 1) {
+    var newCard = deck.pop();
+    hand.push(newCard);
+  }
+  return newCard;
+};
+
+// Display card
+var displayCard = function (hand) {
+  var showCard = [];
+  for (var i = 0; i < hand.length; i += 1) {
+    showCard.push(hand[i].name);
+  }
+  return showCard;
+};
+
+// Calculate score
+var calculateScore = function (hand) {
+  var score = 0;
+  for (var i = 0; i < hand.length; i += 1) {
+    score = score + hand[i].value;
+  }
+  return score;
+};
+
+// Create & shuffle the deck
+var startButton = document.getElementById("start-button");
+startButton.addEventListener("click", function () {
+  deck = [];
```

Comment:
These few lines seem to be repeated several times. Try creating a resetGame function